### PR TITLE
DATA: Add LabCommand

### PR DESCRIPTION
**Pull request by mimendel**

### CODE REVIEW

File: DMX.Core.Api/Migrations/20220815170013_AddLabCommandModel.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+﻿using System;
```
Comment (hassanhabib) [inline DMX.Core.Api/Migrations/20220815170013_AddLabCommandModel.cs:1]: Add copyrights here

Comment (geetika-jain) [inline DMX.Core.Api/Migrations/20220815170013_AddLabCommandModel.cs:1]: Add header

**Comment by AndrewMcClelland** [comment]
Looks like build is passing on this one, so failure on DateTime PR #108 is contained

**Review by hassanhabib — APPROVED** [review]

**Comment by hassanhabib** [comment]
> Looks like build is passing on this one, so failure on DateTime PR #108 is contained

Thanks @AndrewMcClelland I will take a look at that one then.

**Pull request merged**